### PR TITLE
FEATURE: BEND-08 Get ME Endpoint

### PR DESCRIPTION
**Pull request by Kehomaxd**

### CODE REVIEW

**Comment by Kehomaxd** [comment]
@whitehatdevv  what happend here? i dont understand what is wrong

**Comment by whitehatdevv** [comment]
@Kehomaxd you have conflicts, try to fix it pls

**Comment by Kehomaxd** [comment]
@whitehatdevv  done

File: src/http/actions/Account.ts [hunk 1]
```diff
@@ -64,7 +65,102 @@ export module AccountActions {
...
+      path: "work",
+      populate:[
+        {path:"location"}
+      ]
+    }).select (['_id', 'title', 'companyName', 'location', 'actually', 'startDate', 'endDate', 'headline', 'description'])//¿?
+    .populate('location')
+    .populate('education').select(['_id', 'school', 'degree', 'field', 'startDate', 'endDate', 'description'])
+    .exec()
+    .then((dto: any) =>{
+      const dom: Me = {
```
Comment (whitehatdevv) [inline src/http/actions/Account.ts:96]: This object can be created using a mapper. 
```js
f(null, <Me>dto);
```

Please remove all the object transformation.

File: src/models/domain/domain.education.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+export interface Education{
+        id: string;
+        school: string;
+        degree: string;
+        field: string;
+        startDate: Date;
+        endDate: Date;
+        description: string;       
+}
```
Comment (whitehatdevv) [inline src/models/domain/domain.education.ts:14]: Please keep on mind format the code before make a push, it's a mandatory

File: src/models/domain/domain.me.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+import { Direction } from "./domain.direction";
+import { Education } from "./domain.education";
+import { Languages } from "./domain.languages";
+import { Name } from "./domain.name";
+import { PhotoId } from "./domain.photoId";
+import { Works } from "./domain.works";
+
+export interface Me{
+
```
Comment (whitehatdevv) [inline src/models/domain/domain.me.ts:14]: Remove this white space.

**Review by whitehatdevv — CHANGES_REQUESTED** [review]
You have to: 
- Remove all the unused whitespaces.
- Document all the interfaces (DOM objects) -> You have forgotten the author github name
- Fix all this errors.
- Format the code before pushing, select all the code, use your right click and look for an option called format document

**Pull request merged**